### DISCUSSION TITLE
data types present in JavaScript

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#67] [Q&A] [answered]
What are the various data types present in JavaScript?

**Comment by Bunny77K9** [comment, marked as answer]
JavaScript supports the following data types:

    Boolean
    Number
    Object
    Undefined
    Null
    String
    Function

To get in-depth knowledge of Reactjs, Check out our [ReactJS Tutorial](https://intellipaat.com/blog/reactjs-tutorial/)!
